ops/ruby/devops-framework/DEVOPS_FRAMEWORK.md

```diff
@@ -0,0 +1,78 @@
+# DevOps Framework
+
+## Purpose
+This is a small but flexible framework for automating DevOps actions and ensuring they are
+taken in a traceable, repeatable, and consistent way. It operates by  communicating to Google Cloud Platform
```

Comment:
Repeatability and consistency are mentioned here, yet a large divergence between this and the previous system is the lack of docker usage. Was docker considered?